etc/pachd/Dockerfile

```diff
@@ -6,6 +6,7 @@ LABEL name="Pachyderm" \
       vendor="Pachyderm"
 
 COPY --from=pachyderm_build /licenses /
+COPY --from=pachyderm_build /tmp/to-copy /
```

Comment:
It would be good to have a TODO here to clean this up when we are no longer using scratch.